### PR TITLE
confirmation header and note not shown correctly

### LINKED ISSUE #2035: Delete confirmation text not showing
**Describe the bug**
When I delete something for the first time the delete confirmation message is not showing.

**To Reproduce**
Try deleting a goal for the first time:

**Expected behavior**
Delete confirmation as stored in translation files shows.

**Screenshots**
delete_info

**Additional context**
I thought there was already an issue for this, but couldn't find it.

### PR DESCRIPTION
**Pull request by vinaybadgujar102**
resolves #2035

### CODE REVIEW

**Review by tijlleenders — CHANGES_REQUESTED** [review]
Cool!

Can you change the note to "You can always recover this from the Trash." ?

File: src/translations/fr/translation.json [hunk 1]
```diff
@@ -133,7 +133,7 @@
     },
     "delete": {
       "header": "Voulez-vous supprimer cette cible ?",
```
Comment (tijlleenders) [inline src/translations/fr/translation.json:135]:
```suggestion
      "header": "Voulez-vous supprimer cet objectif?",
```

**Review by tijlleenders — APPROVED** [review]
Great!

**Comment by tijlleenders** [comment]
@vinaybadgujar102 Ready for review by Tushar?

File: src/components/GoalsComponents/GoalConfigModal/ConfigGoal.tsx [hunk 2]
```diff
@@ -75,7 +75,8 @@ const ConfigGoal = ({ type, goal, mode }: { type: TGoalCategory; mode: TGoalConf
     });
   }, [goal.id]);
 
-  const [title, setTitle] = useState(goal.title);
+  const goalTitle = t(goal.title);
+  const [title, setTitle] = useState(goalTitle);
```
Comment (Tushar-4781) [inline src/components/GoalsComponents/GoalConfigModal/ConfigGoal.tsx:79]: you can do the above statement directly

**Review by Tushar-4781 — CHANGES_REQUESTED** [review]

**Pull request merged**